### ISSUE TITLE
Discrepancy between horas & missa ranking

### ISSUE THREAD

**Issue by orbanbalage** [DivinumOfficium/divinum-officium#2600]
For 2021-08-22 & Divino Afflatu (possibly others) there was a discrepancy in the rank of the feast between the horas and the missa:

https://www.divinumofficium.com/cgi-bin/horas/officium.pl
```
Immaculati Cordis Beatæ Mariæ Virginis ~ Duplex II. classis
Commemoratio: Dominica XIII Post Pentecosten IV. Augusti
```

and

https://divinumofficium.com/cgi-bin/missa/missa.pl?date=8-22-2021&version=Divino%20Afflatu
```
Dominica XIII Post Pentecosten IV. Augusti ~ Semiduplex Dominica minor
Commemoratio: Immaculati Cordis Beatæ Mariæ Virginis
```

I'm not familiar with the code/logic, but the only difference in the horas/missa files seems to be this:

horas/English/Sancti/08-22.txt:
```
[Rank]
Immaculati Cordis Beatae Mariae Virginis;;Duplex II classis;;5.1;;ex C11

[Rank1960]
Immaculati Cordis Beatae Mariae Virginis;;Duplex II classis;;5;;ex C11
```

vs.

missa/English/Sancti/08-22.txt:
```
[Rank]
Immaculati Cordis Beatae Mariae Virginis;;Duplex II classis;;5;;ex C11
```

In other words, the `horas` seems to be correct, that the feast of the Blessed Virgin is demoted to Commemoration only in the 1960 rubrics (or perhaps in the later rubrics too, but I'm focusing on Divino Afflante at the moment). This same logic is not only not reflected for the `missa`, but there it is lower in rank in all cases.

Perhaps this is pointing at a bigger question - perhaps there are more such discrepancies  in the data elsewhere, that should be detected and corrected. Perhaps this could be mentioned as a condition for code contribution – that the fix should be applies in all files (ie. both `horas` and `missa`), but also across all languages!

If the language variants and maintaned independently from eachother, could someone please confirm this, thank you.

**Comment by APMarcello3** [comment]
Closed by ca3ddaf08. Discussion on this can continue under the "Discussions" concerning the union of the horas and missa files.
